src/Microsoft.Framework.Runtime.Hosting/RuntimeHostBuilder.cs

```diff
@@ -37,13 +38,15 @@ public static RuntimeHostBuilder ForProjectDirectory(string projectDirectory, Nu
                 throw new ArgumentNullException(nameof(runtimeFramework));
             }
 
+            var log = RuntimeLogging.Logger<RuntimeHostBuilder>();
             var hostBuilder = new RuntimeHostBuilder();
 
             // Load the Project
             var projectResolver = new PackageSpecResolver(projectDirectory);
             PackageSpec packageSpec;
-            if (projectResolver.TryResolvePackageSpec(GetProjectName(projectDirectory), out packageSpec))
+            if (projectResolver.TryResolvePackageSpec(GetProjectName(projectDirectory), out packageSpec) && packageSpec != null)
```

Comment:
Yeah, the current code seems to do that if a folder with the specified name exists but doesn't contain a project.json. I think I can fix that.